### PR TITLE
Testnet toggler now follows network state

### PR DESCRIPTION
**Pull request by BreadJS**
## What does this PR address?
When clicking on the testnet toggle in settings it was not following the network state. For example, when it failed to switch to testnet because of a loaded wallet, it still switched the toggle. This has now been fixed right now

## How does this benefit users?
The users can now see the real state of the network with the toggler.

## PIVX Address
`DDS5v6mPqUN3rdzkp3268LV7TKjHCZqy8c`

### CODE REVIEW

File: scripts/global.js [hunk 1]
```diff
@@ -292,6 +292,7 @@ export async function start() {
         domDisplaySettingsBtn: document.getElementById('settingsDisplayBtn'),
         domVersion: document.getElementById('version'),
         domFlipdown: document.getElementById('flipdown'),
+        domTesnetToggler: document.getElementById('tesnetToggler'),
```
Comment (JSKitty) [inline scripts/global.js:295]: Typo: `domTesnetToggler` --> `domTestnetToggler`, the typo is consistent around the PR too.

File: scripts/settings.js [hunk 2]
```diff
@@ -403,6 +406,14 @@ export function toggleTestnet() {
     doms.domGuiBalanceStakingTicker.innerText = cChainParams.current.TICKER;
     doms.domPrefixNetwork.innerText =
         cChainParams.current.PUBKEY_PREFIX.join(' or ');
+
+    // Update testnet toggle in settings
+    if (cChainParams.current.isTestnet) {
+        doms.domTesnetToggler.checked = true;
+    } else {
+        doms.domTesnetToggler.checked = false;
+    }
```
Comment (JSKitty) [inline scripts/settings.js:415]: This code can be shortened to (keep in mind the `Tesnet` typo):
```suggestion
    doms.domTesnetToggler.checked = cChainParams.current.isTestnet;
```

**Review by JSKitty — CHANGES_REQUESTED** [review]
A couple nits.

**Review by JSKitty — APPROVED** [review]
**tACK** 566e121cab9c90347da495a30ca2c4bc14e7c315

Fixed the nits myself, LGTM before v1.0.

**Review by Liquid369 — APPROVED** [review]
tACK 566e121cab9c90347da495a30ca2c4bc14e7c315

**Pull request merged**